data/json/items/gun/shot.json

```diff
@@ -331,53 +391,183 @@
+    "id": "tavor_12",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": "Tavor TS12",
+    "description": "This is a triple tube magazine fed, gas-operated bullpup shotgun of Israeli Weapon Industries make.  It is capable of loading 15 shells, all in a relatively small package.  Like many other modern IWI designs, this looks more like a sci-fi prop gun than a firearm.  An integral top rail is provided for mounting sights.",
+    "weight": "4400 g",
+    "volume": "4040 ml",
+    "looks_like": "ksg",
+    "price": 180000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "wood" ],
+    "dispersion": 320,
     "durability": 6,
-    "clip_size": 8,
-    "reload": 200,
-    "barrel_length": 4,
+    "clip_size": 16,
+    "sight_dispersion": 1000,
+    "ranged_damage": 1,
     "valid_mod_locations": [
       [ "accessories", 2 ],
+      [ "bore", 1 ],
       [ "brass catcher", 1 ],
+      [ "grip mount", 1 ],
+      [ "mechanism", 4 ],
+      [ "muzzle", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights", 1 ],
+      [ "sling", 1 ]
+    ],
+    "flags": [ "RELOAD_ONE" ]
+  },
+  {
+    "id": "USAS_12",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": "USAS 12",
+    "description": "The USAS 12 looks like a boxy upsized caricature of the M16A2.  Like its Auto Assault-12 predecessor, it is a select-fire shotgun fed from large box or drum magazines.  The in-line recoil system and sheer weight make for pleasant shooting.",
+    "weight": "5450 g",
+    "volume": "4695 ml",
+    "looks_like": "saiga_12",
+    "price": 300000,
+    "to_hit": -1,
+    "bashing": 12,
+    "material": [ "steel", "plastic" ],
+    "dispersion": 510,
+    "durability": 6,
+    "barrel_length": 6,
+    "valid_mod_locations": [
+      [ "accessories", 2 ],
+      [ "mechanism", 4 ],
+      [ "barrel", 1 ],
       [ "sling", 1 ],
-      [ "stock", 1 ],
+      [ "muzzle", 1 ],
+      [ "grip", 1 ],
+      [ "rail mount", 1 ],
+      [ "sights mount", 1 ],
+      [ "underbarrel mount", 1 ]
+    ],
+    "modes": [ [ "DEFAULT", "semi-auto", 1 ], [ "AUTO", "auto", 5 ] ],
+    "magazines": [ [ "shot", [ "USAS10mag", "USAS20mag" ] ] ]
+  },
+  {
+    "id": "winchester_1887",
+    "copy-from": "shotgun_base",
+    "type": "GUN",
+    "name": "1887 bootleg shotgun",
+    "description": "One of the first commercially successful repeating shotguns, the Winchester 1887 was specifically made lever-action at Winchester's request.  Though later overshadowed in success by pump designs, the 1887 remains popular today.  This one has a very short barrel, no stock, and would pair nicely with a motorcycle jacket and a Harley Davidson.",
+    "weight": "2994 g",
+    "volume": "1127 ml",
+    "looks_like": "browning_blr",
+    "price": 35300,
+    "to_hit": -1,
+    "bashing": 8,
+    "material": [ "steel", "wood" ],
+    "ranged_damage": -9,
+    "dispersion": 860,
+    "durability": 6,
+    "clip_size": 7,
+    "reload": 120,
+    "skill": "pistol",
```

Comment:
Stealthy Terminator time?
![image](https://user-images.githubusercontent.com/51099123/66662558-0bed1300-ec17-11e9-884c-2326c19c47ba.png)

Reply:
![image](https://user-images.githubusercontent.com/7764202/66672032-f67de600-ec22-11e9-8d41-d2921f9d97c5.png)
Fixed, thanks!

Reply:
![image](https://user-images.githubusercontent.com/7764202/66674270-224f9a80-ec28-11e9-9a11-ad93ae7b31c2.png)

Reply:
Those are the same picture 🤔

Reply:
oh oops